wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/pacgoat/subsystems/Pneumatics.java

```diff
@@ -9,9 +9,8 @@
 
 import edu.wpi.first.wpilibj.AnalogInput;
 import edu.wpi.first.wpilibj.command.Subsystem;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-
 import edu.wpi.first.wpilibj.examples.pacgoat.Robot;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
```

Comment:
Yuck on these.  There's a good reason to separate the examples ones from the rest of wpilib.  Should we instead move the examples to a different package (that's not a subpackage of wpilibj)?

Reply:
As long as they have a common package root, I can easily handle a change in vscode. So yeah a custom package works just fine for me.

Reply:
I think I can put a general suppression on the examples package. Will check when I get home tonight.

Reply:
Seems like these were actually automatically done by intelij when I saved the file and Checkstyle is not checking for it at all. See my latest commit to add enforcement for this.